docs/rackhd/ubuntu_package_installation.rst

```diff
@@ -49,7 +53,45 @@ Ensure Node.js is at version 4.x:
     $ node -v
     v4.4.5
 
-Install the prerequisite packages:
+
+Install & Basically Configure RackHD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ **after Prerequisites installation, there're two options to install and configure RackHD from package**
+ Either (a) or (b) can lead the way to install RackHD from debian packages.
+
+   (a) Install/Configure with Ansible Playbook
+   (b) Install/Configure with Step by Step Guide
+
+
+(a) Install/Configure with Ansible Playbook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+a.1. install git and ansible
+.. code::
+
+  sudo apt-get install  git
+  sudo apt-get install  ansible
+
+a.2. clone RackHD code
+
+.. code::
+
+  git clone https://github.com/RackHD/RackHD.git
+
+a.3. Run the ansible playbooks, it will install the prerequisites packages, install RackHD debian packages, copy default config files..
+
+.. code::
+
+  cd RackHD/packer/ansible
+  ansible-playbook -c local -i "local," rackhd_package.yml
+
+a.4. All the services are upstart and have logs in /var/log/upstart.  Start with 'start on-[something]'  Verify with 'ps | aux | grep node'
```

Comment:
same suggestion - break up what you're setting up as a title and separate content:

`a.4. starting services`

`All the services are upstart and have logs in /var/log/upstart.  Start with 'start on-[something]'  Verify with 'ps | aux | grep node'`